sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -93,24 +99,7 @@ class AdaptiveQueryExecSuite
       assert(smj.size == 1)
       val bhj = findTopLevelBroadcastHashJoin(adaptivePlan)
       assert(bhj.size == 1)
-      checkNumLocalShuffleReaders(adaptivePlan, 2)
-    }
-  }
-
-  test("Change merge join to broadcast join and reduce number of shuffle partitions") {
```

Comment:
This test case is duplicated with the above one now. We add local reader to both sides of broadcast join and won't apply "reduce number of shuffle partitions".